lib/mars_rover.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'coordinate'
+require_relative 'compass'
+require_relative 'current_position'
+class MarsRover
+  attr_accessor :compass, :commands, :coordinate
+
+  def initialize(starting_coordinate_x:, starting_coordinate_y:, starting_direction:, commands: [])
```

Comment:
How would you refactor so that clients can pass a `starting_coordinate` directly instead of `starting_coordinate_x` and `starting_coordinate_y`?